### PR TITLE
#14099: ttnn.clip, clamp interface to follow Pytorch

### LINKED ISSUE #14099: [Bug Report] ttnn.clip interface does not follow Pytorch
**Describe the bug**
`ttnn.clip` has recently changed such that the arguments `min` and `max` are keywords. However, the equivalent arguments for `torch.clip` and also `torch.clamp` and `Hardtanh` are positional instead.

https://pytorch.org/docs/stable/generated/torch.clip.html
```
torch.clip(input, min=None, max=None, *, out=None) → Tensor
```
https://pytorch.org/docs/main/generated/torch.clamp.html
https://pytorch.org/docs/stable/generated/torch.nn.Hardtanh.html#torch.nn.Hardtanh

This changed was observed since `tags/v0.53.0-rc21` but could have been earlier.

**To Reproduce**
```
import ttnn
import torch

tensor = torch.rand((4, 4), dtype=torch.bfloat16)

torch_clip = torch.clip(tensor, 0.5, 0.5)

with ttnn.manage_device(device_id=0) as device:
    ttnn_from_torch = ttnn.from_torch(tensor, layout = ttnn.TILE_LAYOUT, dtype = ttnn.bfloat16, device = device)
    ttnn_clip = ttnn.clip(ttnn_from_torch, -0.5, 0.5)
    ttnn_to_torch = ttnn.to_torch(ttnn_clip)
```

Error message:
```
TypeError: __call__(): incompatible function arguments. The following argument types are supported:
    1. (self: ttnn._ttnn.operations.unary.clip_t, input_tensor: ttnn._ttnn.tensor.Tensor, *, min: Optional[float] = None, max: Optional[float] = None, memory_config: Optional[ttnn._ttnn.tensor.MemoryConfig] = None) -> ttnn._ttnn.tensor.Tensor

```
**Expected behavior**
If the `min` and `max` args are positional like `torch.clip` then `ttnn.clip` would not show any error.

**Please complete the following environment information:**
- OS: Ubuntu 20.04.6 LTS
- tags/v0.53.0-rc21

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
https://github.com/tenstorrent/tt-metal/actions/runs/11474694921 - passed

Work done : To change the float values to positional arguments as requested in #14099

### CODE REVIEW

File: tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py [hunk 1]
```diff
@@ -1288,7 +1288,7 @@ def clip(
     **kwargs,
 ):
     t0 = setup_tt_tensor(x, device, layout[0], input_mem_config[0], dtype[0])
-    t1 = ttnn.clip(t0, min=low, max=high, memory_config=output_mem_config)
+    t1 = ttnn.clip(t0, low, high, memory_config=output_mem_config)
```
Comment (Aswinmcw) [inline tests/tt_eager/python_api_testing/sweep_tests/tt_lib_ops.py:1291]: What happens if I give (t0, high, low)?

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by umadevimcw — APPROVED** [review]

**Review by ayerofieiev-tt — APPROVED** [review]
thank you

**Review by eyonland — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/11514255838 - passed

**Pull request merged**